TwilioVerifySDKTests/TwilioVerify/Sources/Domain/Factor/FactorMapperTests.swift

```diff
@@ -20,7 +20,7 @@
 import XCTest
 @testable import TwilioVerifySDK
 
-// swiftlint:disable force_cast force_try
+// swiftlint:disable force_cast force_try type_body_length
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/1230259fcee173b2bad2b801de0a522beecabdd9/TwilioVerifySDKTests/TwilioVerify/Sources/Domain/Factor/FactorMapperTests.swift#L23"></span>SwiftLint rule 'type_body_length' did not trigger a violation in the disabled region; remove the disable command<br />
<code>superfluous_disable_command</code> <code>FactorMapperTests.swift:23</code></td>
    </tr>
  </tbody>
</table>